.github/workflows/release.yml

```diff
@@ -0,0 +1,33 @@
+name: release
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19.x
```

Comment:
In the spirit of Semver to ensure at least 1.19 is being used  we should be using the 
```
go-version: '>=1.19.0'

Reply:
@andrewpsp good point fixed